shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -83,13 +83,21 @@ - (void)application:(UIApplication*)application
 - (void)userNotificationCenter:(UNUserNotificationCenter*)center
        willPresentNotification:(UNNotification*)notification
          withCompletionHandler:
-             (void (^)(UNNotificationPresentationOptions options))completionHandler
-    API_AVAILABLE(ios(10)) {
-  if (@available(iOS 10.0, *)) {
-    [_lifeCycleDelegate userNotificationCenter:center
-                       willPresentNotification:notification
-                         withCompletionHandler:completionHandler];
-  }
+             (void (^)(UNNotificationPresentationOptions options))completionHandler {
+  [_lifeCycleDelegate userNotificationCenter:center
+                     willPresentNotification:notification
+                       withCompletionHandler:completionHandler];
+}
+
+/**
+ * Calls all plugins registered for `UNUserNotificationCenterDelegate` callbacks.
+ */
+- (void)userNotificationCenter:(UNUserNotificationCenter*)center
+    didReceiveNotificationResponse:(UNNotificationResponse*)response
+             withCompletionHandler:(void (^)(void))completionHandler {
+  [_lifeCycleDelegate userNotificationCenter:center
```

Comment:
This call should be gated by 'respondsToSelector:' since the method is optional.

Reply:
done